pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -2143,7 +2143,7 @@ final: prev:
     meta.homepage = "https://github.com/nvim-lua/diagnostic-nvim/";
   };
 
-  diffview-nvim = buildVimPluginFrom2Nix {
```

Comment:
how does the update logic know vim vs neovim?

Reply:
There is an heuristic on master: if there exists a lua package with the same name then it's a neovim plugin. 
In this MR I've added a whitelist to help me play a bit with better neovim plugin management. In this peculiar case, I've added diffview-nvim to be able to run the neovim check hook.